### DISCUSSION TITLE
Problem with dropdown menu

### DISCUSSION THREAD

**Discussion by roywielens** [twbs/bootstrap#37604] [Q&A] [answered]
The drop-down menu does not work. Who can help me find out what the error is in my code?

```html
					<!-- Header - Nav - Logo -->
					<div class="container-fluid">
						<a class="navbar-brand" href="#">Logo</a>
						
					<!-- Header - Nav - Hamburgermenu -->
					<button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
  					<span class="navbar-toggler-icon"></span>
					</button>
					<div class="collapse navbar-collapse" id="navbarSupportedContent">
						
  						<!-- Header - Nav - Items -->
						<ul class="navbar-nav me-auto mb-2 mb-lg-0">
							<li class="nav-item">
							<a class="nav-link active" aria-current="page" href="#">Home</a>
							</li>
							<li class="nav-item">
							<a class="nav-link" href="#">Aanbod</a>
							</li>
							
								<!-- Header - Nav - Dropdownmenu -->
								<li class="nav-item dropdown">
									<a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">About</a>
								<ul class="dropdown-menu">
									<li><a class="dropdown-item" href="#">Vacancies</a></li>
									<li><a class="dropdown-item" href="#">Our Missions</a></li>
									<li><hr class="dropdown-divider"></li>
									<li><a class="dropdown-item" href="#">Inloggen klanten</a></li>
								</ul>
    							</li>
							
							<!-- Header - Nav - Vervolg - Items-->
    						<li class="nav-item">
								<a class="nav-link disabled">Contact</a>
    						</li>
						</ul>
					</div>
					</div>
				</nav>
			</div>
		</div>
	</div>
	
	<!-- Header - Nav - Search -->
	<div class="container-fluid">
		<div class="row">
			<div class="col">
				<form class="d-flex" role="search">
					<input class="form-control me-2 zoekvenster" type="search" placeholder="Search" aria-label="Search">
					<button class="btn btn-outline-success knop" type="submit">Search</button>
				</form>
			</div>
		</div>
	</div>
</header>
```

**Comment by julien-deramond** [comment, marked as answer]
Hey @roywielens.

You mean the navbar toggler or the dropdown menu within the navbar?
The question being a little bit vague, I'll try to answer anyway.
There's probably a missing `<nav class="navbar navbar-expand-lg bg-light">` surrounding all of this code for having a real navbar with a navbar toggler. If I do that, the dropdown menu is also well rendered and works well.
If you're only talking about the dropdown menu, maybe you don't import the JavaScript Bootstrap correctly 🤷 

If you need a more precise answer, you could try to create a reproducible example by setting up a [Bootstrap CodePen code example](https://codepen.io/team/bootstrap/pen/qBamdLj).

**Reply by roywielens** [reply]
Hello Julien,

Thank you so much for answering this quick. I did mean the dropdown menu. The navbar toggler is working fine.  The only thing not working is the dropdown function. 

I'll try creating a reproducible example by setting up a Bootstrap CodePen code example.

**Reply by julien-deramond** [reply]
I had a bit of time to check this out. I've created https://codepen.io/julien-deramond/pen/PoaVBXZ that embed the code from the discussion. Just added `<nav class="navbar navbar-expand-lg">` surrounding everything to see the actual navbar.
As you can see, the "About" dropdown works fine.

In the settings, I've included Bootstrap CSS minified file and Bootstrap JS minified file (`https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js`):

![Screenshot 2022-12-07 at 13 17 53](https://user-images.githubusercontent.com/17381666/206177534-cdda4ad6-0711-457b-b5e9-7f99473306ff.png)

If it doesn't work in your environment, that's probably because the JS file is not imported correctly.

**Reply by roywielens** [reply]
You are the best!

I entered my code in CodePen and it worked just fine. 
By checking my <script> sources I found out that I had forgotten a source.
